### PR TITLE
fix: strip service account token

### PR DESCRIPTION
**Pull request by cvvz**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
strip service account token to avoid security risk.

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
fix: strip service account token
```

### CODE REVIEW

File: pkg/csi-common/utils.go [hunk 1]
```diff
@@ -111,3 +112,30 @@ func LogGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, h
...
+	if err != nil || parsed == nil {
+		return req.String()
+	}
+
+	volumeContext, ok := parsed["volume_context"].(map[string]interface{})
+	if !ok {
+		return req.String()
+	}
+
+	if _, ok := volumeContext["csi.storage.k8s.io/serviceAccount.tokens"]; !ok {
```
Comment (andyzhangx) [inline pkg/csi-common/utils.go:129]: let's make this func more generic, set `csi.storage.k8s.io/serviceAccount.tokens` as a parameter, thanks.

Comment (andyzhangx) [reply pkg/csi-common/utils.go:129]: rename as `stripSecrect(secretFieldNames []string)`

Comment (cvvz) [reply pkg/csi-common/utils.go]: ok, please take another look

Comment (cvvz) [reply pkg/csi-common/utils.go]: > rename as stripSecrect(secretFieldNames []string)

This will be conflict with the `StripSecrets` function in csi lib

**Review by andyzhangx — APPROVED** [review]
/lgtm
/retest

**Pull request merged**